cql3/statements/raw/truncate_statement.hh

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
+ */
+
+#pragma once
+
+#include "cql3/statements/raw/cf_statement.hh"
+#include "cql3/attributes.hh"
+
+namespace cql3 {
+
+namespace statements {
+
+namespace raw {
+
+class truncate_statement : public raw::cf_statement {
+private:
+    std::unique_ptr<attributes::raw> _attrs;
+public:
+    /**
+     * Creates a new UpdateStatement from a column family name, columns map, consistency
```

Comment:
Copy-pasted comment is incorrect.
To be honest, I don't understand what is this new "raw" truncate statement and how it differs from the regular (cooked? :-)) truncate statement.

Reply:
The difference is in containing `attributes::raw` vs. prepared `attributes`
(and being derived from `raw::cf_statement` vs. keeping a `schema_ptr` member)

Will fix the comment nevertheless. thanks for spotting.

Reply:
fixed in v2